tools/prow-auto-bumper/file.go

```diff
@@ -93,7 +93,7 @@ func (pv *PRVersions) updateAllFiles(fileFilters []*regexp.Regexp, imageFilter *
 	dryrun bool) ([]string, error) {
 	var msgs []string
 	if err := helpers.ChdirToRoot(); err != nil {
-		return msgs, fmt.Errorf("failed to change to root dir")
+		return msgs, fmt.Errorf("failed to change to root dir: '%v'", err)
```

Comment:
```suggestion
		return msgs, fmt.Errorf("failed to change to root dir: '%w'", err)
```

Reply:
Wasn't %w deprecated in go1.14?

Reply:
I heard from you that it's deprecated, but actually seems not? There are still a lot of `%w` in knative/pkg

Reply:
It was failing when we first tried out go1.14, but in beta, might have been fixed

Reply:
done